soc/ite/ec/it8xxx2/linker.ld

```diff
@@ -176,6 +176,9 @@ SECTIONS
 #ifdef CONFIG_SOC_IT8XXX2_SERIAL_IN_RAM
 			*libdrivers__serial.a:*
 #endif
+#ifdef CONFIG_SOC_IT8XXX2_KERNEL_IN_RAM
+			*libkernel.a:*
```

Comment:
Is it possible to include a smaller set of kernel functions than the entire kernel into ILM?  Adding almost 9K to the RAM size is pretty expensive.

Reply:
Here are the reasons for considering the entire libkernel or libzephyr as RAM code:
1. The ILM has increased from 60k in the original IT81xx2 to 256k in the IT82xx2. The main reason for increasing RAM was to account for the potential increase in latency when switching from the legacy RTOS to the Zephyr RTOS. Therefore, in the IT82xx2, there is sufficient space to accommodate the libkernel or libzephyr as RAM code.
2. According to the PD standard specification, `tReceiverResponse`is 15ms. After placing the libkernel and libzephyr as RAM code on the rauru platform, `tReceiverResponse`is reduced from 18ms to 9ms, showing significant improvement and compliance with the specification.
Additionally, the following issues have shown noticeable improvement.
https://partnerissuetracker.corp.google.com/issues/300105688
https://partnerissuetracker.corp.google.com/issues/347824320
3. The libkernel or libzephyr consists of multiple sets. For example, libkernel.a includes `busy_wait.c, device.c, init.c, idle.c, mutex.c, sem.c, system_work_q.c, work.c, thread.c, sched.c, timeout.c, timer.c, poll.c, fatal.c.` Files like `init.c, fatal.c, device.c, etc.,` have minimal impact, but configuring each one individually would result in excessive configurations.

Reply:
Acknowledged.